base/genimport/importer.go

```diff
@@ -269,14 +274,93 @@ func removeAllFilesInDirExcept(o *Output, dir string, except_list []string) {
 }
 
 func createPluginGoModFile(o *Output, pkgpath string, dir string) string {
+	newLines := [][]byte{
```

Comment:
Can we use `x/mod` to write the go.mod as well?  I'm not sure how safe it is to do all this stuff without any attention to quoting (e.g. if your cwd contains spaces).  Plus it should be easier!

Reply:
I'm going to try that as well.